feat(apple): Mention source context (#6462)
getsentry/sentry-docs: src/platform-includes/getting-started-config/apple.mdx

## Provide Debug Information {#debug-symbols}

> Before capturing crashes, you need to provide debug information to Sentry. Furthermore, Sentry can display snippets of your code next to the event stacktraces. This feature is called source context. To enable it use the `include-sources` option when uploading your debug information files. Debug information is provided by <PlatformLink to="/dsym">uploading dSYM files</PlatformLink>.
> 

Reviewer: ```suggestion
To capture crashes, you need to provide debug information to Sentry. You can also use our source context feature to display code snippets next to the event stack traces by enabling the `include-sources` option when uploading your debug information files. Debug information is provided by <PlatformLink to="/dsym">uploading dSYM files</PlatformLink>.

```
